src/test/java/redis/clients/jedis/modules/graph/GraphTransactionTest.java

```diff
@@ -53,7 +53,7 @@ public static void prepare() {
   @Test
   public void testMultiExec() {
```

Comment:
Let's remove since graph is unsupported. We'd be extending bitrot for the future otherwise.

Reply:
@chayim Why remove when it's working?
Moreover, it is testing existing transaction, not the new one.